addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/schema/create-or-update-env-type-config.js

```diff
@@ -28,7 +29,7 @@ const schema = {
       type: 'string',
       maxLength: 128,
       minLength: 2,
-      pattern: '^[A-Za-z0-9-_ ]+$',
+      pattern: nameRegex,
```

Comment:
use `idRegex` for the id pattern in this file! Add test coverage for id in env-type-service.create()

Reply:
fixed